specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2020-02-07-preview/examples/SapMonitors_PatchTags.json

```diff
@@ -0,0 +1,40 @@
+{
+  "parameters": {
+    "api-version": "2020-02-07-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "myResourceGroup",
+    "sapMonitorName": "mySapMonitor",
+    "tagsParameter": {
+      "tags": {
+        "testkey": "testvalue"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.HanaOnAzure/sapMonitors/mySapMonitor",
+        "name": "mySapMonitor",
+        "type": "Microsoft.HanaOnAzure/sapMonitors",
+        "location": "westus",
+        "tags": {
+          "testkey": "testvalue"
+        },
+        "enableCustomerAnalytics": true,
+        "logAnalyticsWorkspaceArmID": "",
+        "logAnalyticsWorkspaceID": "",
+        "logAnalyticsWorkspaceSharedKey": "",
+        "monitorProviders": [
+          {
+            "name": "myHanaProviderInstance",
+            "type": "hana",
+            "properties": "{\"hostname\":\"10.0.0.10\",\"dbName\":\"SYSTEMDB\",\"sqlPort\":30015,\"dbUsername\":\"SYSTEM\",\"dbPassword\":\"PASSWORD\"}"
+          }
+        ],
```

Comment:
So PATCH actually returns the whole thing even if just the tags are changed?
Also, remove the `dbPassword` from the response

Reply:
that is my understanding, yet